docs/core/compatibility/unsupported-apis.md

```diff
@@ -0,0 +1,319 @@
+
+| Member | Platform |
+| - | - |
+| <xref:System.Configuration.RsaProtectedConfigurationProvider?displayProperty=nameWithType> (all members) | All |
+
+## System.Console
+
+| Member | Platform |
+| - | - |
+| <xref:System.Console.Beep?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Console.BufferHeight?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Console.BufferWidth?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Console.CursorSize?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Console.CursorVisible?displayProperty=nameWithType> (get only) | Linux and macOS |
+| <xref:System.Console.MoveBufferArea%2A?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Console.SetWindowPosition%2A?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Console.SetWindowSize%2A?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Console.Title?displayProperty=nameWithType> (get only) | Linux and macOS |
+| <xref:System.Console.WindowHeight?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Console.WindowLeft?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Console.WindowTop?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Console.WindowWidth?displayProperty=nameWithType> (set only) | Linux and macOS |
+
+## System.Data.Common
+
+| Member | Platform |
+| - | - |
+| <xref:System.Data.Common.DbDataReader.GetSchemaTable%2A?displayProperty=nameWithType> (throws <xref:System.NotSupportedException>) | All |
+
+## System.Diagnostics.Process
+
+| Member | Platform |
+| - | - |
+| <xref:System.Diagnostics.Process.MaxWorkingSet?displayProperty=nameWithType> (set only) | Linux |
+| <xref:System.Diagnostics.Process.MinWorkingSet?displayProperty=nameWithType> (set only) | Linux |
+| <xref:System.Diagnostics.Process.ProcessorAffinity?displayProperty=nameWithType> | macOS |
+| <xref:System.Diagnostics.Process.MainWindowHandle?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Diagnostics.Process.Start%2A?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Diagnostics.ProcessStartInfo.UserName?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Diagnostics.ProcessStartInfo.PasswordInClearText?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Diagnostics.ProcessStartInfo.Domain?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Diagnostics.ProcessStartInfo.LoadUserProfile?displayProperty=nameWithType> | Linux and macOS |
+| <xref:System.Diagnostics.ProcessThread.BasePriority?displayProperty=nameWithType> (set only) | Linux and macOS |
+| <xref:System.Diagnostics.ProcessThread.BasePriority?displayProperty=nameWithType> (get only) | macOS |
+| <xref:System.Diagnostics.ProcessThread.ProcessorAffinity?displayProperty=nameWithType> (set only) | Linux and macOS |
+
+## System.IO
```

Comment:
so the following API is an interesting case. It always throws for Linux and macOS on a specific circumstance
https://docs.microsoft.com/en-us/dotnet/api/system.io.memorymappedfiles.memorymappedfile.createnew?view=netcore-3.1#System_IO_MemoryMappedFiles_MemoryMappedFile_CreateNew_System_String_System_Int64_System_IO_MemoryMappedFiles_MemoryMappedFileAccess_

not sure if we should add here or not

Reply:
I had that documented but removed it in [this commit](https://github.com/dotnet/docs/pull/16387/commits/900bf90ade1b1a1c230a52c72f35ad4cb8202d78) since it doesn't always throw.

Reply:
But I don't actually even see that PNSE exception [in the source code](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.cs), so I'm wondering if it is old information in the API docs that should be updated. @carlossanlop ?

Reply:
@gewarren @mairaw this information is correct.

The [CreateNew](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.cs#L271) method internally calls a method called `CreateCore` which is split by platform. The [Windows version of that method](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.Windows.cs#L20) does the job without problem, but [the Unix one throws PNSE](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.Unix.cs#L32).

Reply:
Thanks for checking Carlos. I won't document it then, a) because it doesn't always throw and b) because it throws a different exception (CreateNamedMapsNotSupportedException).

Reply:
Ahh, good to know. So even though the name makes it sound like a variant of PNSE, it's not the exception we currently care about.

Reply:
Well, when it actually throws, it is PNSE, no?
https://github.com/dotnet/runtime/blob/master/src/libraries/System.IO.MemoryMappedFiles/src/System/IO/MemoryMappedFiles/MemoryMappedFile.Unix.cs#L119

Reply:
Cirrect. Unix is not a supported platform. I don't know why we throw that specific exception but I think its purpose is similar to PNSE.